tests/__helpers/testApp.ts

```diff
@@ -43,7 +43,12 @@ export const testApp = (settings: Settings) => {
         const prettierConfigPath = require.resolve('../../.prettierrc')
         const content = await typegenFormatPrettier(prettierConfigPath)(source, type)
 
-        return content.replace("'@nexus/schema'", "'../../..'")
+        return content.replace("'@nexus/schema'", "'../../../src'")
+      },
+      features: {
+        abstractTypes: {
+          resolveType: true,
```

Comment:
Any particular reason we don't want to use the default for the integration tests?

Reply:
This just made me aware of some ambiguity in the settings. I don't think it is documented. Which is: Is this a merge semantic or an overwrite semantic where all the other defaults are `false`?

That is:

```ts
// default
{
  abstractTypes: {
    isTypeOf: true,
    __typename: false,
    resolveType: false,
  }
}
```
Now given input:
```ts
{
  abstractTypes: {
    resolveType: true,
  }
}
```
Does that mean (A):
```ts
{
  abstractTypes: {
    isTypeOf: true,
    __typename: false,
    resolveType: true,
  }
}
```
Or (B):
```ts
{
  abstractTypes: {
    isTypeOf: false,
    __typename: false,
    resolveType: true,
  }
}
```

Overall I like that with B its "what you see is what you get". I can see though how this wouldn't generalize to all cases of settings. And thus would A be surprising behaviour? (Sure we'd mention it in jsdoc...)